src/app/share/views/SharedLinksView/SharedView.tsx

```diff
@@ -121,7 +121,14 @@ export default function SharedView(): JSX.Element {
 
   useEffect(() => {
     if (!currentFolderId && !isShareDialogOpen && !folderUUID) {
-      setTimeout(fetchRootFolders, 200);
+      setTimeout(() => {
+        setPage[0];
```

Comment:
The function is not being executed here. `setPage[0]` -> `setPage(0);`
In `onShowInvitationsModalClose` is the same error, change it in this PR please.

Reply:
Done